src/pfe/file-watcher/server/src/projects/projectUtil.ts

```diff
@@ -1233,8 +1248,20 @@ export async function runScript(projectInfo: ProjectInfo, script: string, comman
     const logName = getLogName(projectInfo.projectID, projectInfo.location);
     const logDir = await logHelper.getLogDir(projectInfo.projectID, projectInfo.projectName);
     const projectName = path.basename(projectInfo.location);
-    let args = [projectInfo.location, LOCAL_WORKSPACE, projectID, command, containerName, String(projectInfo.autoBuildEnabled), logName, projectInfo.startMode,
-        projectInfo.debugPort, "NONE", logDir];
+    let args = [
+        projectInfo.location,
+        LOCAL_WORKSPACE,
+        projectID,
+        command,
+        containerName,
+        String(projectInfo.autoBuildEnabled),
+        logName,
+        projectInfo.startMode,
+        projectInfo.debugPort,
+        "NONE",
+        logDir,
+        projectInfo.imagePushRegistry,
```

Comment:
Did you mean to add projectInfo.imagePushRegistry here? (It wasn't present before but this may be ok or it may be a cut and paste error.)